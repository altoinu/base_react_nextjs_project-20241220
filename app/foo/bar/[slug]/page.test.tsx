import Page from "./page";
import { render, screen } from "@testing-library/react";

describe("bar page", () => {
  it("should render", () => {
    const { container: PageContainer } = render(
      <Page params={{ slug: "hello!" }} />,
    );

    expect(PageContainer).toMatchSnapshot();
  });

  it("should render a heading", () => {
    render(<Page params={{ slug: "hello!" }} />);
    const heading = screen.getByRole("heading");
    expect(heading.textContent).toBe("Slug: hello!");
  });
});
