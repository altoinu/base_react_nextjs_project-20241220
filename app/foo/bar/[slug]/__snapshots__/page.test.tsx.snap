// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bar page should render 1`] = `
<div>
  <h3>
    Slug: 
    hello!
  </h3>
</div>
`;
